packages/client/src/components/SafeSettings/components/Owners/AddSafeOwner.js

```diff
@@ -36,50 +46,57 @@ const AddSafeOwner = ({ onCancel, onNext, safeOwners }) => {
   };
 
   const nextButtonClasses = [
-    "button flex-1 is-primary",
-    addressValid ? "" : "disabled",
+    "button flex-1 is-primary ",
+    addressValid ? "has-text-weight-bold" : "disabled",
   ];
+  const addrInputClasses = [
+    "p-4 rounded-sm column is-full is-size-6",
+    addressBorderClass
```

Comment:
not a big deal but prob better to derive this from `addressValid` and `address.length` vs using state as every setState causes a re-render